src/content/en/2024/structured-data.md

```diff
@@ -2,17 +2,611 @@
+![A year on year comparison of Microformats2 usage on desktop pages][image24]
+![A year on year comparison of microformats2 usage][image25]
+
+![A year on year comparison of Microformats usage on mobile pages][image26]
+
+![A year on year comparison of Microformats usage on desktop pages][image27]
+
+Cross-device implementation remains relatively consistent, though with some variation between mobile and desktop. The data shows a **general decline in traditional Microformats usage** from 2022 to 2024, particularly in **review-related properties** like `hReview` and `hReview-aggregate`. This decline reflects the **industry’s shift toward more modern structured data formats** like JSON-LD and RDFa, which offer **broader functionality** and better **integration with current web standards**.
+
+Despite this decline, **Microformats** and **Microformats2** remain useful in specific contexts where lightweight, human-readable semantic data is needed. However, their overall presence continues to be eclipsed by more versatile formats like JSON-LD, which dominate the structured data landscape.
+
+![A year on year comparison of Microformats usage on desktop pages][image28]
+
+## Microdata
+
+**Microdata** continues to be widely used for **structural elements** and **navigation data**, particularly within legacy platforms and sites where simpler, static page structures are required. The most frequently implemented types include `schema.org/webpage` (appearing on **8.34% of mobile pages**) and `schema.org/sitenavigationelement` (used on **6.42% of mobile pages**), indicating the format’s enduring relevance for **webpage structure** and **site navigation**.
+
+![A year on year comparison of Microdata usage on mobile pages][image29]
+
+**Navigation-related types** like `listitem` and `breadcrumblist` have also seen steady growth, reflecting the need for more organized and structured navigation data, particularly on mobile devices. However, **content-specific types** such as `schema.org/article` and `schema.org/product` remain less common, with adoption rates of **1.77%** and **1.50%** respectively, as developers increasingly turn to JSON-LD for more flexible and scalable implementations.
+
+![A year on year comparison of microdata usage][image30]
+
+While **Microdata** remains a significant format for **fundamental webpage structure**, its use in **dynamic content** and **e-commerce** applications has been gradually overtaken by more modern formats like **JSON-LD**, which offer broader support for content enrichment and structured data scaling across large websites.
+
+## JSON-LD
+
+**JSON-LD types** continue to show diverse implementation patterns across websites, with the **WebSite** schema leading adoption at **12.73%** of mobile pages, significantly higher than other types. **Organization** and **LocalBusiness** types maintain a strong presence at **7.16%** and **3.97%**, respectively, reflecting their importance in establishing **entity identity**.
```

Comment:
What does "diverse implementation patterns" mean? We should be explicit. For example:

> JSON-LD is widely used across the websites we analyzed, with varied types of data implemented from WebSite to Organization.